bundles/core/src/main/java/com/adobe/cq/commerce/core/cacheinvalidation/internal/InvalidateCacheEventListener.java

```diff
@@ -44,6 +45,9 @@ public class InvalidateCacheEventListener implements EventListener {
 
     private Session session;
 
+    @Reference(target = "(|(" + ServiceUserMapped.SUBSERVICENAME + "=" + InvalidateCacheSupport.SERVICE_USER + ")(!(subServiceName=*)))")
```

Comment:
In (subServiceName=*) also subServiceName could be replaced with ServiceUserMapped.SUBSERVICENAME ?
You can use static imports to make all that more concise.